app/components/modals/cropper-modal.js

```diff
@@ -5,8 +5,13 @@ export default class extends ModalBase {
     let viewport = {};
     let factor = 150;
     const aspectRatio = this.getWithDefault('aspectRatio', [2, 1]);
-    viewport.width = aspectRatio[0] * factor;
-    viewport.height = aspectRatio[1] * factor;
+    if (this.session.currentRouteName === 'public.cfs.edit-speaker' || this.session.currentRouteName === 'public.cfs.new-speaker') {
```

Comment:
That is a really bad logic. A component should have no knowledge of the current route. It should be decoupled from the route. Use props for changing the behaviour of component

Reply:
@kushthedude can you help here ? As i think now there would be need of two croppers now.

Reply:
No